packages/gitbook-docs/src/markdoc/schema/link.ts

```diff
@@ -18,6 +18,16 @@ export const link: Schema = {
   },
   transform: (node, config) => {
     const attrs = node.transformAttributes(config);
+
+    /* eslint-disable functional/no-expression-statements */
+    /* eslint-disable functional/immutable-data */
+    if (attrs.href && attrs.href.includes('mailto:')) {
+      // delete everything before the 'mailto:'
+      attrs.href = attrs.href.replace(/.*mailto:/, 'mailto:');
+    }
+    /* eslint-enable functional/no-expression-statements */
+    /* eslint-enable functional/immutable-data */
```

Comment:
```suggestion
    const attrs = pipe(node.transformAttributes(config), (attributes) =>
      pipe(
        attributes.href && attributes.href.includes('mailto:'),
        b.fold(
          () => attributes,
          () => ({
            ...attributes,
            href: attributes.href.replace(/.*mailto:/, 'mailto:'),
          })
        )
      )
    );
```
an alternative way keeping data immutable

Reply:
That's not working, tests does not pass anymore with that modification, also, i think that 'b' is not the most meaningful name to give to something